### ISSUE TITLE
redirect lpt1 on com in dosbox x 

### ISSUE THREAD

**Issue by iw2evk** [joncampbell123/dosbox-x#4441]
### Is your feature request related to a problem? Please describe.

Hi,
t's possible to redirect the lpt1 port on a setted com ports?

Thanks

Roberto
Milan

### What you want

Example. I've dosbox x on ubuntu 20.04.
i've setted 2 usb port lie com 1 and com2.
Now i want to redirect a reallpt (i've attached a eprom programmer on usb via usb to lpt adapter) on com1
It's possible add this ?

### Describe alternatives you've considered

No alternative

### Additional information

notebook lenovo T420 8gb ram 
No lpt port no com ports only 3 usb port tipe 2

### Have you checked that no similar feature request(s) exist?

- [X] I have searched and didn't find any similar feature request.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by rderooy** [comment]
You can have a look at the wiki, there is a page on printing which mentions the parallel port options.
https://dosbox-x.com/wiki/Guide%3ASetting-up-printing-in-DOSBox%E2%80%90X

But... While I have never tested this myself (I don't have any parallel ports or devices left), I consider the likelihood that this will work to be low.

If you can get it to work, it would probably be with the "realport" option, which you would have to point to the host parallel port. You would also have to ensure that permissions are set on the device port that your user can actually read and write to it (add your user to the "lp" group as mentioned on the wiki page should do the trick).

Do not bother with the "file dev:..." option, it will be uni-directional, and not suitable for a bi-directional device such as an eeprom programmer.

**Comment by dbjh** [comment]
I agree with everything @rderooy wrote. I would like to add that if you cannot make the device /dev/parport0 appear (or any /dev/parport followed by a number), possibly after issuing the command "modprobe parport_pc" as user root, it will not work. There appears to be a misconception (or unclarity in the documentation) that the parallel port pass-through code works with any device name. It does not. The code can talk with the device driver ppdev, which only creates devices that start with /dev/parport.
@rderooy: Contributing to the parallel port documentation is still high on my TODO list.

**Comment by iw2evk** [comment]
Hi ,
and redirecting in dos using Mode command?
Redirect printing:
      MODE LPTn[:]=COMm[:]
May be work ?

**Comment by dbjh** [comment]
As I wrote, if you don't have /dev/parport0 it will not work. Have you checked whether that device is present?
I would not know how your command (inside DOSBox-X) could possibly work, but you tell me :-) Just try it and see for yourself.
